include/zcbor_decode.h

```diff
@@ -46,33 +46,33 @@ bool zcbor_uint_decode(zcbor_state_t *state, void *result_uint, size_t uint_size
 /** The following applies to all _expect() functions that don't have docs.
  *
  * @param[inout] state   The current state of the decoding.
- * @param[in]    result  The expected value.
+ * @param[in]    value   The expected value.
  *
  * @retval true   If the result was decoded correctly and has the expected value.
  * @retval false  If the decoding failed or the result doesn't have the
  *                expected value.
  */
 /** Consume and expect a pint/nint with a certain value. */
-bool zcbor_int32_expect(zcbor_state_t *state, int32_t result);
-bool zcbor_int64_expect(zcbor_state_t *state, int64_t result);
-bool zcbor_uint32_expect(zcbor_state_t *state, uint32_t result);
-bool zcbor_uint64_expect(zcbor_state_t *state, uint64_t result);
-bool zcbor_size_expect(zcbor_state_t *state, size_t result);
+bool zcbor_int32_expect(zcbor_state_t *state, int32_t value);
+bool zcbor_int64_expect(zcbor_state_t *state, int64_t value);
+bool zcbor_uint32_expect(zcbor_state_t *state, uint32_t value);
+bool zcbor_uint64_expect(zcbor_state_t *state, uint64_t value);
+bool zcbor_size_expect(zcbor_state_t *state, size_t value);
 
 /** Consume and expect a pint/nint with a certain value, within a union.
  *
  * Calls @ref zcbor_union_elem_code then @ref zcbor_[u]int[32|64]_expect.
  */
-bool zcbor_int32_expect_union(zcbor_state_t *state, int32_t result);
-bool zcbor_int64_expect_union(zcbor_state_t *state, int64_t result);
-bool zcbor_uint32_expect_union(zcbor_state_t *state, uint32_t result);
-bool zcbor_uint64_expect_union(zcbor_state_t *state, uint64_t result);
+bool zcbor_int32_expect_union(zcbor_state_t *state, int32_t value);
+bool zcbor_int64_expect_union(zcbor_state_t *state, int64_t value);
+bool zcbor_uint32_expect_union(zcbor_state_t *state, uint32_t value);
+bool zcbor_uint64_expect_union(zcbor_state_t *state, uint64_t value);
 
 /** Decode and consume a bstr/tstr */
 bool zcbor_bstr_decode(zcbor_state_t *state, struct zcbor_string *result);
-bool zcbor_bstr_expect(zcbor_state_t *state, struct zcbor_string *result);
+bool zcbor_bstr_expect(zcbor_state_t *state, struct zcbor_string *value);
```

Comment:
why is it not considered a result here and below?

Reply:
_expect() functions take the expected value as input instead of the result as output. This is true even here, where the argument is a pointer.